back/config/schemas/settings.schema.json.erb

```diff
@@ -1115,6 +1115,18 @@
           "allowed": { "type": "boolean", "default": false },
           "enabled": { "type": "boolean", "default": false }
         }
+      },
+
+      "permissions_custom_fields": {
+        "type": "object",
+        "title": "Permission custom fields on project actions",
+        "description": "Enables the option for admins to add custom questions to project action permissions.",
+        "additionalProperties": false,
+        "required": ["allowed", "enabled"],
+        "properties": {
+          "allowed": { "type": "boolean", "default": false },
+          "enabled": { "type": "boolean", "default": false }
+        }
```

Comment:
@jamesspeake I did the BE changes as well by following a previous PR. I think it's correct, but let me know if I've missed anything :)

Reply:
Is there another setting somewhere to enable it by default for new premium tenants maybe? :thinking: Not quite sure how that works.

Reply:
Ah ok - I thought maybe that was `back/engines/commercial/multi_tenancy/db/seeds/tenants.rb`? It's not being enabled by default in that file, but if that's for actual production platforms I'll disable it by default instead then :)

Reply:
Yes you're absolutely correct, those are the seeds for the local env - I didn't actually look at the files you changed - just read the comment! All good then

Reply:
Ok great!! Thank you! :)